### DISCUSSION TITLE
Stuck on 7.1.5 version

### DISCUSSION THREAD

**Discussion by pierrevano** [squidfunk/mkdocs-material#2789] [Q&A] [answered]
Hi,

I'm trying to upgrade to the latest 7.1.9 version to add Google Analytics 4 but I'm facing some issues.

Did the 3 following commands to upgrade to the latest versions:
```
pip3 install --upgrade pip
pip3 install --upgrade mkdocs
pip3 install --upgrade mkdocs-material
```

When I'm checking the **mkdocs-material** version with ```pip3 show mkdocs-material``` I've got this:
```
Name: mkdocs-material
Version: 7.1.9
Summary: A Material Design theme for MkDocs
Home-page: https://squidfunk.github.io/mkdocs-material/
Author: Martin Donath
Author-email: martin.donath@squidfunk.com
License: MIT
Location: /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages
Requires: mkdocs, markdown, Pygments, mkdocs-material-extensions, pymdown-extensions
Required-by: mkdocs-material-extensions
```

But when I look in the path ```mkdocs-material/src/partials/integrations/``` I still got the 7.1.5 structure.
And the base.html file still indicates that I'm still on 7.1.5:
```
<meta name="generator" content="mkdocs-{{ mkdocs_version }}, mkdocs-material-7.1.5">
```

As a result I'm not able to add the new GA4 support. Any steps or debug I'm missing? Thanks!

**Comment by squidfunk** [comment, marked as answer]
IMHO there are two possibilities:

- Different installation locations of `mkdocs` and `mkdocs-material`, possibly through a virtual environment. Check with `which mkdocs` and make sure both live in the same location.
- You overrode `base.html`

 > But when I look in the path `mkdocs-material/src/partials/integrations/` I still got the 7.1.5 structure.
And the base.html file still indicates that I'm still on 7.1.5:

The `src` directory is not bundled with the `pip` installation, so it could also be that you're using a local copy of the repository.

**Reply by pierrevano** [reply]
Thanks @squidfunk for the reply! As I supposed it was coming from an installation issue, I started from scratch with a proper python3 install (and not a git clone) and I managed to make it work. Thanks
